scrapers/errata.py

```diff
@@ -68,7 +69,49 @@ def update_neo4j(self, advisories):
                 'type_': advisory['type'],
                 'update_date': advisory['update_date'],
                 'updated_at': advisory['updated_at']
-            })[0]
+            }
+
+            for associated_build in self.get_associated_builds(advisory['id']):
+                # If this is set, that means it was once part of the advisory but not anymore.
+                # This relationship needs to be deleted if it exists.
+                build = KojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+                container_adv = False
+                if build:
+                    if build.__label__ == 'ContainerKojiBuild':
+                        adv = self.create_container_build_advisory(adv_params)
+                        container_adv = True
+                    else:
+                        adv = Advisory.create_or_update(adv_params)[0]
+
+                if associated_build['removed_index_id']:
+                    if build:
+                        adv.attached_builds.disconnect(build)
+                else:
+                    # This key shouldn't be stored in Neo4j
+                    del associated_build['removed_index_id']
+                    # Trouble teiid only if the build is not present in Neo4j
+                    if build:
+                        if container_adv:
+                            is_container_build = True
```

Comment:
I thought an advisory could have multiple types of attached builds, so you can't just assume it's a container build every time.

Reply:
I confirmed  this with MikeB and it seems that is not the case. We had a bug which was giving us conflicting results.